pkg/apis/v1alpha1/dnspolicy_types.go

```diff
@@ -37,6 +44,12 @@ type DNSPolicySpec struct {
 
 	// +optional
 	LoadBalancing *LoadBalancingSpec `json:"loadBalancing"`
+
+	// +required
+	// +kubebuilder:validation:Enum=simple;loadbalanced
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="RoutingStrategy is immutable"
+	// +kubebuilder:default=loadbalanced
+	RoutingStrategy RoutingStrategy `json:"routingStrategy"`
```

Comment:
routingStrategy is required, but having the default means you don't have to specify it and will default to "loadbalanced" (The current behaviour)